cannon/mipsevm/multithreaded/mips.go

```diff
@@ -143,6 +143,22 @@ func (m *InstrumentedState) handleSyscall() error {
 	case exec.SysOpen:
 		v0 = exec.SysErrorSignal
 		v1 = exec.MipsEBADF
+	case exec.SysClockGetTime:
+		v0 = 0
+		v1 = 0
```

Comment:
We should. I've limited it emulation to realtime (used for logging) and monotonic clocks. The VM returns an error for any other clock.

Reply:
And fwiw, I ran this on the op-program and confirmed that it uses only realtime and monotonic clocks. So we shouldn't observe any other clocks used by the program. But just in case that changes, I'm opting use errno to indicate that they're unsupported rather than panic.